universe/org.themoviedb/index.js

```diff
@@ -33,14 +34,16 @@ module.exports = class MovieClass extends Tp.BaseDevice {
             return Promise.all(parsedResponse.results.map(async (result) => {
                 const castQuery = `https://api.themoviedb.org/3/movie/${result.id}/credits?api_key=${this.constructor.metadata.auth.api_key}&language=en-US`;
                 let id = new Tp.Value.Entity(String(result.id), result.title);
-                let oneDate = new Date(Date.now());
                 if ((result.release_date !== undefined) && (String(result.release_date) !== ''))
                     oneDate = new Date(result.release_date);
+                else
+                    oneDate = undefined;
                 const movieObj = {
                     id,
                     description: result.overview,
                     release_date: oneDate,
                     rating_score: Number(result.vote_average),
+                    genres: result.genre_ids,
```

Comment:
You should map the genre IDs to an array of Entity objects containing the genre names as well, otherwise the agent won't be able to speak out the genre name.